### ISSUE TITLE
"Address for your connection to the server" does not give the correct address as an option

### ISSUE THREAD

**Issue by girlpunk** [mregni/EmbyStat#1358]
# Bug report

**Describe the bug**
During the setup wizard, users are prompted to "Choose the correct address for your connection to the server". However, depending on the environment Emby and/or EmbyStat is being run from, the correct address may not be available.

**To Reproduce**

1. Run both Emby and EmbyStat from Docker containers
3. Complete the setup wizard of EmbyStat, to the point where you are asked to "choose the correct address for your connection to the server

**Expected behavior**
The user is able to enter the correct address

**Screenshots**
In some situations, only internal Docker names or IPs are available

**System (please complete the following information):**

- OS: Docker on Linux
- Browser Firefox
- Version 0.2.0-beta.23
- Media server type Jellyfin

**Additional context**
It is likely that Emby and EmbyStat will not be able to dectect this correctly, as such this should probably be a text box that's prefilled with whatever address EmbyStat is using to connect.

**Comment by mregni** [comment]
Hi, thanks for logging this bug.
You mean that the value you entered in the wizard is not available in the "Address for your connection to the server" dropdown?

**Comment by girlpunk** [comment]
The address that EmbyStat connects on is different to the one I connect on. The first is available in the drop-down, but the latter is not.

**Comment by mregni** [comment]
![image](https://user-images.githubusercontent.com/22617019/92448969-a6542e80-f1b9-11ea-8d0f-060675e5e0a5.png)
Create a small diagram so I can understand you clearly 😄 

Do you mean that the address used by EmbyStat to connect to Jellyfin (x) is different than the calls that are made from your client browser? Meaning that EmbyStat (frontend code) needs to use a different URL than the backend. If so, maybe it's a better idea to route everything over the EmbyStat backend instead of calling Jellyfin directly from the browser.

**Comment by girlpunk** [comment]
Yes, that's entirely correct. In my case, x would be `http://jellyfin:8086` and z would be `https://jellyfin.my.domain:443`.

Calling Jellyfin from the browser isn't nessasaraly bad, and the user-accessable address (z) would still be needed if EmbyStat was to link to Jellyfin or specific content in it.

For reference, y would be `https://embystat.my.domain:443`. The address for Jellyfin to access EmbyStat (i.e. X but in the opposite direction) would also be different to Y, but neither of these appear to be settings anywhere in EmbyStat.

**Comment by mregni** [comment]
Oke this will be a new feature instead of a bug because the current dropdown is used for setting up the x address and I presumed that x and z would always be the same. Seems I didn't think of this so will try to include this in the next beta.
